README.md

```diff
@@ -1,56 +1,32 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# BookStore
 
+> A Bookstore app that adds and removes a user's book
 
 ## Built With
 
-- Major languages
+- HTML, CSS, JS
 - Frameworks
 - Technologies used
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
 ## Authors
 
-👤 **Author1**
+👤 **Promise Okechukwu**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@promzzykoncepts](https://github.com/promzzykoncepts)
+- Twitter: [@promzzy](https://twitter.com/promzzy)
+- LinkedIn: [Promise Okechukwu](https://linkedin.com/in/promiseokechukwu)
 
 👤 **Author2**
 
```

Comment:
- [ ] Please add your member's name because he/she would contribute equally in your project so you should take care of these thing :+1: